### ISSUE TITLE
Firestore as database

### ISSUE THREAD

**Issue by santoshgistto** [webiny/webiny-js#513]
## This is: Question
Is it possible use firebase firestore as database ?

**Comment by doitadrian** [comment]
Hello @santoshgistto,

our whole system is based on MongoDB and, in the Webiny cloud, on AWS Document DB.
So the most natural choice would be to use these databases when talking about custom development with Webiny.

But if you really wanted to use Firebase for your app, it can be done, since in a GraphQL resolver you can basically write anything you need to get the job done. Although I wouldn't advise it since you get a lot of functionality out of the box by using the default setup.

**Comment by santoshgistto** [comment]
Thanks @doitadrian for response 
i am trying connect database using mongodb scalegrid.io provider
its failing 
logs :
`(node:95341) UnhandledPromiseRejectionWarning: Error: connect ECONNREFUSED 172.107.181.211:27017
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1191:14)
(node:95341) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:95341) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.`

**Comment by doitadrian** [comment]
Hi @santoshgistto , if you ask me, it seems your connection params are incorrect. It's an error sent from your DB provider. Can you double check? Are you sure your DB provider even allows connections from a local machine?

**Comment by santoshgistto** [comment]
Thanks @doitadrian  it was my issue, network is not allowing

**Comment by santoshgistto** [comment]
I have another question, is there any way to publish static page instead of providing client app ?

**Comment by Pavel910** [comment]
If you are using the CMS - then no, everything is a single-page-app.
However, our Cloud handles the prerendering for crawlers and bots, so from the SEO perspective you don't need to worry about it, crawlers will get the fully loaded content.

**Comment by santoshgistto** [comment]
Thanks @Pavel910  for response, how about support `gatsbyjs` ? it supports `graphql`

**Comment by Pavel910** [comment]
@santoshgistto I haven't tried deploying a `gatsbyjs` built app to Webiny Cloud, but I think it should work. Webiny apps are plain CRA (create-react-app) apps so there is no reason it shouldn't work.

In the meantime, we've implemented SSR (server side render) and we are releasing it shortly so the problem with SEO and page load time is now solved for good. Your visitors will be receiving full HTML so there will be no need to use `gatsbyjs` if static HTML is your only concern.
